src/Workspaces/CSharp/Portable/SyntaxDifferencing/CSharpSyntaxDifferenceService.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.SyntaxDifferencing;
+
+namespace Microsoft.CodeAnalysis.CSharp.SyntaxDifferencing
+{
+    [ExportLanguageService(typeof(SyntaxDifferenceService), LanguageNames.CSharp), Shared]
+    internal class CSharpSyntaxDifferenceService : SyntaxDifferenceService
+    {
+        public override string Language => LanguageNames.CSharp;
+
+        public override SyntaxMatch ComputeBodyLevelMatch(SyntaxNode oldBody, SyntaxNode newBody)
+        {
+            return new SyntaxMatch(StatementSyntaxComparer.Instance.ComputeMatch(oldBody, newBody));
```

Comment:
Calculating match for method bodies is not quite this simple. See
http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp.Features/EditAndContinue/CSharpEditAndContinueAnalyzer.cs,328776d844a6284e